Mu2eG4/geomMacros/clean_volume_names.C

```diff
@@ -0,0 +1,18 @@
+//
+// The volume names in the CSV can contain memory address (i.e. 0xAAAAAAA).
+// For comparing geometries these need to be removed
+//
+
+std::string clean_volume_names(std::string name) {
+  bool all_found = false;
+  while (!all_found) {
+    auto pos_0x = name.find("0x");
+    if (pos_0x == std::string::npos) {
+      all_found = true;
+    }
+    else {
+      name = name.erase(pos_0x, 9); // always 9 chars
+    }
+  }
```

Comment:
Wouldn't it be simpler to just find the first instance of 0x and delete from there to the end of the string?  I think that "always 9" might be an accident of the machine/OS/compiler we are using now.  You could also strength the test to 0x followed by all digits?    C has an isdigit function.  I am not sure if there is something better in C++.

Reply:
Thanks, Rob. The reason I had it this way was because I was also passing the volume's path (since I wanted to do a diff of two geometries). But you raise a good point so in 4adcb16d9, I've made a separate ```clean_path_names``` function, which calls the ```clean_volume_names``` function that now just deletes to the end of the string

Reply:
Thanks.